src/api.cc

```diff
@@ -443,90 +483,56 @@ std::vector<TufTarget> toTufTargets(const std::vector<Uptane::Target>& targets)
   return ret;
 }
 
+// Throws BundleMetaError
+Json::Value AkliteClient::getBundleMeta(const LocalUpdateSource* local_update_source) const {
```

Comment:
For example, if we pass the `client_` and `tuf_repo_` as input params then this can become non-class static method.

Reply:
Done